lib/manager/pyenv/extract.ts

```diff
@@ -0,0 +1,12 @@
+import * as datasourceDocker from '../../datasource/docker';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile {
+  const dep: PackageDependency = {
+    depName: 'python',
+    currentValue: content.trim(),
+    datasource: datasourceDocker.id,
```

Comment:
I think docker is a bad datasource, where did pyenv gets the python version from?

Reply:
https://github.com/renovatebot/renovate/issues/9757#issuecomment-827915385